examples/10_streaming_write.cpp

```diff
@@ -1,4 +1,9 @@
-#define BUILD_STREAMING_EXAMPLE false
+#ifdef _WIN32
```

Comment:
Is this a general windows issue or a problem with MSVC?

Reply:
[Just with MSVC](https://github.com/ornladios/ADIOS2/blob/5948ca8a85e05eaf2ff07c6b64fa049fe0c4f9bb/cmake/DetectOptions.cmake#L320). Should I just drop the check and do a better check in the ADIOS2 backend?

Reply:
Yes, that would be better I think.
Let's also add a link to that place as a comment.

Reply:
done